src/screen/Council/MotionsScreen.tsx

```diff
@@ -107,67 +101,48 @@ function Motion({item}: {item: DeriveCollectiveProposal}) {
   };
 
   return (
-    <View style={motionStyle.container}>
-      <View style={motionStyle.mainRow}>
-        <Text category={'h4'}>{formatNumber(votes?.index)}</Text>
-        <TouchableOpacity onPress={() => setOpen(!open)} style={motionStyle.titleContainer}>
-          <Text category={'p1'} style={motionStyle.title} numberOfLines={1}>
-            {section}.{method}
-          </Text>
-          <Icon
-            name={open ? 'arrow-up-outline' : 'arrow-down-outline'}
-            style={globalStyles.icon}
-            fill={theme['color-basic-600']}
-          />
-        </TouchableOpacity>
-        {/*<Text>{formatNumber(votes?.threshold)}</Text>*/}
-        <Text category={'c1'}>{`Aye ${votes?.ayes.length}/${membersCount} `}</Text>
-        <Padder scale={0.5} />
-        {(() => {
-          if (data?.isMember) {
-            if (isCloseable) {
-              return (
-                <View>
-                  <Button status={'warning'} size={'tiny'} onPress={onPressClose}>
-                    Close
-                  </Button>
-                </View>
-              );
-            } else if (isVoteable) {
-              return (
-                <View style={motionStyle.buttons}>
-                  <Button status={'danger'} size={'tiny'} onPress={onPressNay}>
-                    Nay
-                  </Button>
-                  <Padder scale={0.5} />
-                  <Button status={'success'} size={'tiny'} onPress={onPressAye}>
-                    Aye
-                  </Button>
-                </View>
-              );
-            }
-          }
-        })()}
-      </View>
-      {open ? (
-        <View style={motionStyle.footer}>
-          <Text category={'c1'} style={[motionStyle.desc, {color: theme['color-basic-600']}]}>{`${formatCallMeta(
-            meta,
-          )}`}</Text>
-          <Padder scale={1} />
-          <CallInspector call={proposal} />
-        </View>
-      ) : null}
-    </View>
+    <Proposal
+      proposal={proposal}
+      accessoryLeft={() => {
+        return <Text category={'h4'}>{formatNumber(votes?.index)}</Text>;
```

Comment:
Will this be different in other context?

Reply:
yes we are using Proposal also in ReferendaScreen. there we are showing the index of the proposal as accessoryLeft.